### PR TITLE
Complete sprint modal window

### PR DESCRIPTION
**Pull request by FeloniukM**
image

### CODE REVIEW

File: backend/Tasque.Core.WebAPI/Controllers/SprintController.cs [hunk 1]
```diff
@@ -6,13 +6,22 @@
 
 namespace Tasque.Core.WebAPI.Controllers;
 
-[Route("api/sprint")]
+[Route("api/sprint/")]
```
Comment (Anatolii-Ivanov) [inline backend/Tasque.Core.WebAPI/Controllers/SprintController.cs]: This change not needed

File: frontend/src/app/components/complete-sprint/complete-sprint-dialog/complete-sprint-dialog.component.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+@Component({
+  selector: 'app-complete-sprint-dialog',
+  templateUrl: './complete-sprint-dialog.component.html',
+  styleUrls: ['./complete-sprint-dialog.component.sass']
+})
+export class CompleteSprintDialogComponent implements OnInit {
+
+  @Input() sprint: SprintInfo = { id: -1, name: '', openIssue: 0, complatedIssues: 0 };
+
+  public finishBtnName = 'Complate sprint';
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/components/complete-sprint/complete-sprint-dialog/complete-sprint-dialog.component.ts]: Compl**e**te

File: frontend/src/app/components/complete-sprint/complete-sprint-dialog/complete-sprint-dialog.component.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import { SprintService } from 'src/core/services/sprint.service';
+
+@Component({
+  selector: 'app-complete-sprint-dialog',
+  templateUrl: './complete-sprint-dialog.component.html',
+  styleUrls: ['./complete-sprint-dialog.component.sass']
+})
+export class CompleteSprintDialogComponent implements OnInit {
+
+  @Input() sprint: SprintInfo = { id: -1, name: '', openIssue: 0, complatedIssues: 0 };
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/components/complete-sprint/complete-sprint-dialog/complete-sprint-dialog.component.ts:14]: Let's add at least one line commnets what each @Input and @Output means

File: frontend/src/app/components/complete-sprint/complete-sprint.component.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+export class CompleteSprintComponent implements OnInit {
+
+  public completeBthClass = 'fill gray';
+
+  constructor(public matDialog: MatDialog) { }
+
+  ngOnInit(): void {
+  }
+
+  OpenCompSprintDialog(): void {
```
Comment (oleksandr-vovk) [inline frontend/src/app/components/complete-sprint/complete-sprint.component.ts]: for method name should be used lowerCamelCase

Comment (FeloniukM) [reply frontend/src/app/components/complete-sprint/complete-sprint.component.ts]: I fixed it

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by andpii — APPROVED** [review]

**Pull request merged**